ts/Core/Chart/Chart.ts

```diff
@@ -664,29 +641,112 @@ class Chart {
     }
 
     /**
-     * Order all series above a given index. When series are added and ordered
-     * by configuration, only the last series is handled (#248, #1123, #2456,
-     * #6112). This function is called on series initialization and destroy.
+     * Insert a series or an axis in a collection with other items, either the
+     * chart series or yAxis series or axis collections, in the correct order
+     * according to the index option and whether it is internal. Used internally
+     * when adding series and axes.
+     *
+     * @private
+     * @function Highcharts.Chart#insertItem
+     * @param  {Highcharts.Series|Highcharts.Axis} item
+     *         The item to insert
+     * @param  {Array<Highcharts.Series>|Array<Highcharts.Axis>} collection
+     *         A collection of items, like `chart.series` or `xAxis.series`.
+     * @return {number} The index of the series in the collection.
+     */
+    public insertItem(
```

Comment:
Q: Would it make sense to move this function to Utilities? Since there's no need for the context I'm not sure if we need to keep it in Chart class.

Perhaps, with some minor refactoring in force-directed-graph I could use it in the future too (instead of `addElementsToCollection` and `removeElementFromCollection`)

Reply:
Sure, will do 👍